### DISCUSSION TITLE
Getting started. Using VScode extension. Imixs-BPMN runtime?

### DISCUSSION THREAD

**Discussion by hwood-vivup** [imixs/open-bpmn#307] [Q&A]
Brand new user here. Think I need some "getting started" help 😬
I installed Open-BPMN extension in VS Studio code. It works!... after I realised I needed to install java!

I tried to move onto next steps in the docs but things seem a bit different from the description. I guess I can skip the eclipse workbench page https://www.imixs.org/doc/modelling/eclipse_workbench.html. but then do I need to "make sure that your current eclipse project is assigned to the Imixs-BPMN runtime"  https://www.imixs.org/doc/modelling/create_model.html Perhaps not, so then I skip it, but then e.g. here: https://www.imixs.org/doc/modelling/process.html. it talks about "Workflow" properties which I'm not seeing. Is that because I didn't set-up the link (from VSCode) to the Imixs-BPMN runtime?

**Comment by rsoika** [comment]
Hi  @hwood-vivup, thanks for you interest in our project.
First of all: we migrate currently form the [Eclipse-BPMN2 Project](https://eclipse.dev/bpmn2-modeler/) to [Open-BPMN](https://www.open-bpmn.org/). Both projects produce BPMN 2.0 Models.

The Imixs-Workflow Project makes use of the BPMN 2.0 extension mechanism. This means, a BPMN model provides additional information beside the graphical representation.  Both the old [Imixs Eclipse based Modeller](https://github.com/imixs/imixs-bpmn) as also the new [Imixs-Open-BPMN Modeller ](https://github.com/imixs/imixs-open-bpmn) are using this extension mechanism.

As you have already installed the Open-BPMN VS-Code extension you are on the right track. You can find a general overview how to model with Imixs Workflow [here](https://www.imixs.org/doc/modelling/howto.html).

What you are looking for are the extension properties. On the bottom of the Diagram plane you find a property panel that you can resize. 

![image](https://github.com/imixs/open-bpmn/assets/1339124/6367f7d9-ecaf-4e27-a647-9717085bae92)


And in this property panel you find all the additional properties that can be used and interpreted by Imixs-Workflow. To activate these extensions you simply need to add the "Imixs-Worklfow Extension" from the Tool Palette into a Task or CatchEvent. Next you will see the new features. This extension mechanism is totally open and you can implement your own extensions too.

You can find example models in the [Imixs-Process-Manager](https://github.com/imixs/imixs-process-manager) project. Take a look into the /src/workflow/ folder [here](https://github.com/imixs/imixs-process-manager/tree/master/src/workflow).

The documentation on [Imixs.org](https://www.imixs.org/doc/modelling/process.html) is still valid, even if some dialogs have changed layout.

I hope this will help you.


===
Ralph
